modules/@apostrophecms/migration/index.js

```diff
@@ -231,41 +231,36 @@ module.exports = {
       // Perform the actual migrations. Implementation of
       // the @apostrophecms/migration:migrate task
       async migrate(options) {
-        await self.apos.lock.lock(self.__meta.name);
```

Comment:
All I did here was remove the lock because I moved it up a level. Removing the try block resulted in github thinking this is very exciting but it's not.